templates/registration/password_reset_email.html

```diff
@@ -1,17 +1,16 @@
 <span style="display:block;margin: 0 2px;padding: 1em;border: 3px solid #FFDE05;background-color: #000;border-radius: 6px;font-size: .95em;color: #444;margin-bottom:0.75em">
 <div style="display:table;margin-left:2em"><img src="https://avatars1.githubusercontent.com/u/6934864?v=3&s=101" style="display:inline;vertical-align: middle">
-<h1 style="font-size:4em;display:inline;vertical-align: middle"><a href="//{{ domain }}" style="text-decoration:none;color:gray"><span style="color: #FFDE05">DM::</span>OJ</a>
+<h1 style="font-size:4em;display:inline;vertical-align: middle"><a href="{{ protocol }}://{{ domain }}" style="text-decoration:none;color:gray"><span style="color: #FFDE05">DM::</span>OJ</a>
       </h1>
 </div>
 </span>
 <div style="display:block;margin: 0 2px;padding: 1em;border: 3px solid #2980B9;background-color: #f8f8f8;border-radius: 6px;font-size: .95em;color: #444;">
 
-{% trans username=user.get_username() %}
-<b>Forgot your password on the {{ site_name }}? Don't worry!</b><br><br>
-To reset the password for your account "{{ username }}", click the button below.
-{% endtrans %}
+<b>{% trans %}Forgot your password on the {{ site_name }}? Don't worry!{% endtrans %}</b>
```

Comment:
Could we change this to something like
```
{{ _('Forgot your password on the %(site_name)s? Don't worry!', site_name=site_name) }}
```
just to match with the line below.

Reply:
Done. I updated the pr's description too.